bgc_data/combined_bgc_data.sql

```diff
@@ -31,31 +31,31 @@ tss_avg AS (
 trip_depths AS (
    SELECT 
       ppl."TripCode",
-      ppl."Depth_m",
+      ppl."SampleDepth_m",
       ppl."SampleID"
    FROM picoplankton_avg ppl
 UNION
    SELECT 
       tss."TripCode",
-      tss."Depth_m",
+      tss."SampleDepth_m",
       tss."SampleID"
    FROM tss_avg tss
 UNION
    SELECT
       pig."TripCode",
-      pig. "Depth_m",
+      pig. "SampleDepth_m",
       pig."SampleID"
    FROM bgc_pigments_data pig 
 UNION
    SELECT 
       che."TripCode",
-      che."Depth_m"::text,
+      che."SampleDepth_m"::text,
       che."SampleID"
    FROM bgc_chemistry_data che
 )
    SELECT
       bm.*,
-      td."Depth_m",
+      td."SampleDepth_m",
```

Comment:
Note that this will require changes in the geoserver-config, as the [query for this layer](https://github.com/aodn/geoserver-config/blob/4f2fcd96011efe51b44d809c70344758e9247364/workspaces/imos/JNDI_imos_bgc_db/combined_bgc_data/featuretype.xml#L45) is being sorted using this field.
Some of the others will be the same - please check!

Reply:
Good point @mhidas 
I'll have a look and then put a PR that I suppose we need to merge after this one to avoid Geoserver to break?